From 96d5b5eba1f806a42351defeddab9734e9d6546b Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Fri, 19 May 2017 00:06:48 +0800
Subject: [PATCH 1811/5242] MLK-14947-4 ARM: dts: set usb1 vbus power polarity
 for imx8qm arm2

commit  11848ce95eb26acdea260a05c622290d0e4d9198 from
https://source.codeaurora.org/external/imx/linux-imx.git

Set the OTG1 vbus power polarity to be active high on
fsl-imx8qm-lpddr4-arm2.dts.

Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 480391f..a555d7d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -213,6 +213,7 @@
 	srp-disable;
 	hnp-disable;
 	adp-disable;
+	power-polarity-active-high;
 	disable-over-current;
 	status = "okay";
 };
-- 
1.7.9.5

